app/controllers/course/assessment/submission/answer/forum_post_response/posts_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+class Course::Assessment::Submission::Answer::ForumPostResponse::PostsController < \
+  Course::Assessment::Submission::Answer::Controller
+  
+  # TODO: add access control
+
+  def selected
```

Comment:
@aylz would suggest keeping domain / business logic in the models, rather than exposing that logic here. For example the forum post answer model can provide a method to fetch all the necessary posts. This way it is much easier to write tests (especially for the model); and when there are future schema or data changes you need to (mostly) change the front end and model files, and not need to touch the controllers.

Reply:
thanks for the suggestion! will be fixing that!